src/Server.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE RankNTypes    #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Server (api, app) where
+
+import Data.Proxy                 (Proxy (..))
+
+import Servant
+
+import Servant.API.Generic
+import Servant.Server.Generic
+
+data Routes route = Routes {
+    _get :: route :- "api" :> "the-answer" :> Get '[JSON] Int
```

Comment:
We won't have an underscore in the beginning. Instead, it should start with the data type name (without Site suffix), then some meaning, and after should have `Route` suffix. So, in this case it should be `hintmanGetRoute`.